### ISSUE TITLE
[functional testing] Remove DEBUG log level output

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#3698]
## Description

The test output from a failed Selenium functional test provides a lot of log output at the INFO and DEBUG levels, e.g.:

```
connectionpool.py          435 DEBUG    "GET / HTTP/1.1" 200 6210
connectionpool.py          264 INFO     Starting new HTTP connection (1): localhost
connectionpool.py          395 DEBUG    Setting read timeout to None
connectionpool.py          435 DEBUG    "GET / HTTP/1.1" 302 219
connectionpool.py          310 INFO     Resetting dropped connection: localhost
connectionpool.py          395 DEBUG    Setting read timeout to None
connectionpool.py          435 DEBUG    "GET /login HTTP/1.1" 200 3866
```

This is rarely useful and it clutters up the test output significantly. The output also includes a base64 encoded Firefox profile. While this DEBUG output is also in place on `develop`, the size of the Firefox profile is much smaller (only a few lines), whereas on `tbb-0.9.0` it is a massive wall of base64 encoded text. We should resolve this to ensure that the test output is useful. 

Note that since we are using Selenium 3+ and geckodriver on `tbb-0.9.0` (we use selenium<3 on `develop`) and as such the method to resolve this is likely different between `tbb-0.9.0` and `develop`, we should resolve in the `tbb-0.9.0` branch first.

Epic: #3659

**Comment by msheiny** [comment]
i got a fix here https://github.com/freedomofpress/securedrop/commit/a1f013408721aa4f87bb3fb8439e9f21d7e2adfc

**Comment by redshiftzero** [comment]
resolved by #3817
